Fst2List.cpp

```diff
@@ -1499,6 +1918,7 @@ int CFstApp::getWordsFromGraph(int &changeStrToIdx, unichar changeStrTo[][MAX_CH
       listOut = 1;
       exploreSubAuto(1);
       if (verboseMode) {
+      	u_printf("Verbose\n");
```

Comment:
Is this a debug message? If not, please use the right indentation.

Reply:
It was
done